app/controllers/application_controller.rb

```diff
@@ -14,6 +14,7 @@ class ApplicationController < ActionController::Base
 
   helper_method :decorated_session, :reauthn?, :user_fully_authenticated?
 
+  prepend_before_action :add_new_relic_trace_attributes
```

Comment:
Should this go below `prepend_before_action :set_locale`? If there's a possibility that adding the New Relic trace attributes crashes, the locale won't be set in the app and might result in the same issue we saw on TTP launch day where people were seeing the language of the app change.

Reply:
I'd argue it's more important for debugging to have the request ID passed through in New Relic traces no matter what, since this is the only thing that allows us to match up a New Relic error trace with the corresponding ELB access log and nginx access log entries. Doesn't seem like it makes a huge difference in any case.